lib/ecto/migration/runner.ex

```diff
@@ -110,6 +127,8 @@ defmodule Ecto.Migration.Runner do
   Checks if a table or index exists.
   """
   def exists?(object) do
+    execute_commands()
```

Comment:
This is the only function that implicitly triggers executing of all commands. Thought it would be okay to do this since it seems like it's only used in tests. Let me know if it should be changed.